packages/E2ETest/windows/ReactUWPTestApp/ReactUWPTestApp.csproj

```diff
@@ -160,7 +160,7 @@
       <Version>6.2.9</Version>
     </PackageReference>
     <PackageReference Include="XamlTreeDump">
```

Comment:
Would be great to have the infrastructure code for RNW in our repo instead of a personal repo.